src/server/controller/metadata/export/importAll.ts

```diff
@@ -0,0 +1,9 @@
+import { ExportedTableData } from 'meta/metadata/export'
+
+import { BaseProtocol, DB } from 'server/db/db'
+
+import { importTable } from './importTable'
+
+export const importAll = async (allData: Array<ExportedTableData>, client: BaseProtocol = DB): Promise<void> => {
+  await Promise.all(allData.map((tableData) => importTable(tableData, client)))
```

Comment:
I also think, that there will be foreign keys order to respect. so an order should be respected when importing